lib/net/authorize/api/controller/ARBCancelSubscriptionController.php

```diff
@@ -3,13 +3,14 @@
 
 use net\authorize\api\contract\v1\AnetApiRequestType;
 use net\authorize\api\controller\base\ApiOperationBase;
+use net\authorize\util\Log;
 
 class ARBCancelSubscriptionController extends ApiOperationBase
 {
-    public function __construct(AnetApiRequestType $request)
+    public function __construct(AnetApiRequestType $request, Log $logger = null)
```

Comment:
If the $logger is an instance of \net\authorize\util\Log class, how will it support any external log libraries?

Reply:
The `\net\authorize\util\Log` class currently does not have a PSR-3 interface that it implements. This repo in general has not really aimed for PSR standards to this point, however if you like I can work to refactor the `\net\authorize\util\Log` class, making this a major version bump as it would have backwards incompatible changes. Is that the direction you would like this to move?